qiskit/circuit/library/standard_gates/ecr.py

```diff
@@ -75,6 +75,8 @@ class ECRGate(Gate):
                     -i  & 1   &  0  & 0
                 \end{pmatrix}
     """
+    gate_name = "ecr"
```

Comment:
`name` might be preferable (so that this could more naturally extend to instructions), and maybe slightly shorter (since most of the gates are already named `*Gate`).

Reply:
✅ done :)